backend/geonature/core/gn_meta/routes.py

```diff
@@ -259,13 +253,12 @@ def uuid_report():
     id_import = params.get("id_import")
     id_module = params.get("id_module")
 
-    query = DB.session.query(Synthese).select_from(Synthese)
-
-    if id_module:
-        query = query.filter(Synthese.id_module == id_module)
-
-    if ds_id:
-        query = query.filter(Synthese.id_dataset == ds_id)
+    query = (
+        DB.select(Synthese)
+        .select_from(Synthese)
+        .where_if(id_module, Synthese.id_module == id_module)
```

Comment:
```suggestion
        .where_if(id_module is not None, Synthese.id_module == id_module)
```
L’`id_module` peut être 0 (généralement le cas pour le module GEONATURE)

Reply:
J'ajoute ça